### ISSUE TITLE
Ididid

### ISSUE THREAD

**Issue by ramazansen** [ramazansen/politurkcem#1]
﻿{
  "language": "English",
  "startmenu.new_game": "Yenı öğünİ",
  "startmenu.multiplayer": "MULTIPLAYER",
  "startmenu.resume_game": "RESUME GAME",
  "startmenu.settings": "Settings",
  "startmenu.high_score": "High Score",
  "startmenu.throne_room": "Throne Room",
  "startmenu.about": "About",
  "news.title": "POLYTOPIA NEWS",
  "news.description": "Welcome!\n This is where we share the latest news from the Polytopia universe. Stay tuned!",
  "gamemodepicker.title": "GAME MODE",
  "tribepicker.title": "PICK YOUR TRIBE",
  "tribepicker.categories.humantribes": "Regular Tribes",
  "tribepicker.categories.specialtribes": "Special Tribes",
  "tribepicker.categories.specialtribes.description": "A collection of tribes that are a bit out of this world...",
  "tribepicker.categories.random": "Let fate decide!",
  "tribepicker.categories.random.button": "RANDOM TRIBE",
  "tribepicker.categories.random.selected.title": "Alakazam!",
  "tribepicker.categories.random.selected.text": "Random tribe selected",
  "tribepicker.restore": "restore purchases",
  "tribepicker.restoring": "restoring...",
  "tribepicker.reset": "reset purchases",
  "tribepicker.tba": "TBA",
  "tribepicker.underconstruction": "Under Construction",
  "tribepicker.underconstruction.description": "We are still working on inventing this tribe. Growing fruits, developing languages and architecture. That takes time you know. Follow @Midjiwan on Instagram or Twitter and you will be the first to know when new tribes arrive!",
  "tribepicker.freetribe": "Free Tribe",
  "tribepicker.freetribe.description": "This tribe is available for free and cannot be purchased to enable online multiplayer.",
  "tribepicker.taken": "Taken",
  "tribepicker.enable": "ENABLE",
  "tribepicker.disable": "DISABLE",
  "tribepicker.disabled": "disabled",
  "tribepicker.disabled.description": "Tribe disabled, it cannot be used by you or the AI.",
  "tribepicker.pick": "PICK",
  "tribepicker.yourname": "Your name",
  "tribepicker.anonymous": "Anonymous",
  "tribepicker.firstplayer": "First player must be a human",
  "tribepicker.pickyour": "Pick your tribe",
  "tribepicker.playertype": "Player type",
  "tribepicker.news.readmore": "Read more...",
  "tribepicker.toprating": "Top rating {0}%",
  "tribepicker.toprating.next": "{0}% needed for next star",
  "tribepicker.topscore": "Top score {0}",
  "tribepicker.topscore.next": "{0} for next star",
  "tribepicker.players": "{0} players",
  "tribepicker.mapsize": "Map Size: {0} tiles",
  "tribepicker.gamemode": "Game Mode: {0}",
  "gamesettings.title": "GAME SETUP",
  "gamesettings.yourname": "Your name",
  "gamesettings.anonymous": "Anonymous",
  "gamesettings.gamename": "Game name",
  "gamesettings.game": "Game {0}",
  "gamesettings.players": "Players",
  "gamesettings.opponents": "Opponents",
  "gamesettings.unlockmore": "Unlock more tribes to play with more opponents",
  "gamesettings.notavailable": "Not Available",
  "gamesettings.info.multiplayer": "{0} players, {1} tiles map",
  "gamesettings.info.local": "{0} opponents, {1} tiles map",
  "gamesettings.info.turnlimit30": ", 30 turn limit",
  "gamesettings.info.difficulty.bonus": "Difficulty bonus: {0}%",
  "gamesettings.difficulty": "Difficulty",
  "gamesettings.difficulty.easy": "Easy",
  "gamesettings.difficulty.normal": "Normal",
  "gamesettings.difficulty.hard": "Hard",
  "gamesettings.difficulty.crazy": "Crazy",
  "gamesettings.startgame": "START GAME",
  "gamesettings.creatingworld": "CREATING WORLD",
  "gamesettings.mode": "Game Mode",
  "gamesettings.createslot": "Creating game slot...",
  "gamesettings.createslot.error": "Error creating game",
  "gamesettings.createslot.error.info": "Make sure you are online and try again.",
  "gamesettings.size": "Map Size",
  "gamesettings.size.tiny": "Tiny",
  "gamesettings.size.normal": "Normal",
  "gamesettings.size.large": "Large",
  "gamesettings.size.disabled": "Not Available",
  "gamesettings.network": "Network",
  "gamesettings.network.online": "Online",
  "gamesettings.network.passplay": "Pass & Play",
  "gamesettings.online.disabled": "Online is locked",
  "gamesettings.online.disabled.info": "There are some things you need to fix to play online multiplayer",
  "gamesettings.online.info": "Play with your friends online using our multiplayer server.",
  "gamesettings.passplay.info": "Play with your friends offline on this device by passing it around.",
  "gamesettings.size.tiles": "{0} tiles map.",
  "gamesettings.continue": "CONTINUE",
  "gamemode.perfection.caps": "PERFECTION",
  "gamemode.perfection": "Perfection",
  "gamemode.perfection.description.button": "Show your skills on the global hiscore in the classic 30 turns game.",
  "gamemode.perfection.description": "Get the highest possible score before the time runs out.",
  "gamemode.perfection.win": "We have reached the end of time. The memory of your tribe will echo in eternity!",
  "gamemode.perfection.loss": "We have reached the end of time.",
  "gamemode.domination.caps": "DOMINATION",
  "gamemode.domination": "Domination",
  "gamemode.domination.description.button": "Play until there is only one tribe left, with no time limit.",
  "gamemode.domination.description": "Wipe all the other tribes from the face of the square. There can be only one.",
  "gamemode.domination.win": "You have defeated all the other tribes and unified the entire square!",
  "gamemode.domination.loss": "Your last city was lost, you have been defeated.",
  "gamemode.glory.caps": "GLORY",
  "gamemode.glory": "Glory",
  "gamemode.glory.description": "First to reach {0} points win",
  "gamemode.glory.win": "The {0} points score was reached!",
  "gamemode.might.caps": "MIGHT",
  "gamemode.might": "Might",
  "gamemode.might.description": "Capture all capitals to win",
  "gamemode.might.win": "All capitals were captured",
  "gamemode.death": "Your last city was lost, you have been defeated.",
  "world.intro.title": "Mighty Leader!",
  "world.intro.text": "You have been chosen to rule the {0} tribe. Explore the world and expand your empire, but watch out for the other tribes.",
  "world.intro.objective": "Objective: {0}",
  "world.turn.end": "End Turn",
  "world.turn.end.question": "End your turn?",
  "world.turn.end.confirm": "Confirm",
  "world.turn.next": "Next Turn",
  "world.turn.finish": "Finish Game",
  "world.turn.nomoves": "No more moves available, end turn",
  "world.turn.start": "START",
  "world.turn.exit": "Exit",
  "world.turn.waiting": "Waiting for {0} to play...",
  "world.turn.waiting.unknown": "Waiting for an unknown tribe to play...",
  "world.turn.ready": "Tap save when ready",
  "world.turn.your": "Your turn",
  "world.turn.remaining": "{0} turns left",
  "world.turn.last": "Last turn!",
  "world.turn.replaying": "Replaying...",
  "world.unit.info.from": "From the city of {0}.",
  "world.unit.veteran": "This unit is a veteran.",
  "world.unit.veteran.progress": "{0}/{1} to become a veteran.",
  "world.unit.ability": "Unit Ability",
  "world.unit.health": "Health",
  "world.unit.attack": "Attack",
  "world.unit.defence": "Defence",
  "world.unit.movement": "Movement",
  "world.unit.range": "Range",
  "world.unit.disembark.title": "Leave {0}",
  "world.unit.disembark.message": "Disembarking this unit will dismantle your {0}. Do you wish to proceed?",
  "world.unit.evolve": "Your unit has grown to a {0}!",
  "world.unit.evolve.title": "Unit Growth",
  "world.unit.dissolved": "Your unit has dissolved",
  "world.building.info": "The city of {0}",
  "world.building.village": "Village",
  "world.building.capture.ready": "Will be ready to capture next turn",
  "world.building.capture.ready.title": "Entering {0}!",
  "world.building.capture.warning": "Capturing will be completed next turn if you don't stop them.",
  "world.building.capture.warning.title": "{0} is under siege!",
  "world.attract.sanctuary": "Your sanctuary has attracted a wild animal!",
  "world.loading": "LOADING {0}%",
  "world.suggestion.title": "Here's a Tip!",
  "world.suggestion.message": "You should {0}",
  "world.suggestion.disable": "(These suggestions can be switched off in the menu)",
  "world.ranks": "1st,2nd,3rd,4th,5th,6th,7th,8th,9th,10th,11th,12th",
  "world.road.connected.title": "New trade route!",
  "world.road.connected.message": "{0} is now connected to your capital!",
  "world.tech.new.title": "New Tech!",
  "world.tech.new.message": "You have discovered the secret of {0}",
  "world.reward.levelup": "{0} level up!",
  "world.reward.building": "You can now construct {0}! This epic monument will bring wealth and glory to the nearest city.",
  "world.reward.building.title": "{0} was completed",
  "world.meet.tribe": "You meet {0}",
  "world.task.new": "You got a new Task!",
  "tribes.nature": "Nature",
  "tribes.xin-xi": "Xin-xi",
  "tribes.xin-xi.info": "They start their journey in the dense mountains, surrounded by beautiful cherry blossom.\n\n{0} starts the game with the '{1}' tech.",
  "tribes.imperius": "Imperius",
  "tribes.imperius.info": "Huge mountains and green valleys. The {0} climate is perfect for growing fruit. \n\n{0} starts the game with the '{1}' tech.",
  "tribes.bardur": "Bardur",
  "tribes.bardur.info": "Surviving the harsh eternal winter of the {0} woods is not an easy task, but {0} seems to thrive here.\n\n{0} starts the game with the '{1}' tech.",
  "tribes.oumaji": "Oumaji",
  "tribes.oumaji.info": "The seemingly endless, sun blessed desert is the home of the {0} tribe.\n\n{0} starts the game with the '{1}' tech.",
  "tribes.kickoo": "Kickoo",
  "tribes.kickoo.info": "White sandy beaches with coconut palms. Abundance of fruit and fish. Welcome to the home of the {0}. \n\n{0} starts the game with the '{1}' tech.",
  "tribes.hoodrick": "Hoodrick",
  "tribes.hoodrick.info": "The yellow autumn leaves of the {0} woods are perfect hideouts for its peculiar mushroom stuffing inhabitants. \n\n{0} starts the game with the '{1}' tech.",
  "tribes.luxidoor": "Luxidoor",
  "tribes.luxidoor.info": "The {0} love expensive things, jewels, rare spices and exclusive ornaments. That is also why they are the most expensive tribe in Polytopia!\n\nTheir citizens lead very comfortable lives, draped in the finest purple silk. But will they survive outside the walls of their beloved capital?  \n\n{0} starts the game with a huge walled capital.",
  "tribes.vengir": "Vengir",
  "tribes.vengir.info": "Frowned upon by the other tribes and pushed into the unpleasant wastelands. Will they tolerate this injustice or rise to fight back?  \n\n{0} start the game with the '{1}' tech and a powerful Swordman.",
  "tribes.zebasi": "Zebasi",
  "tribes.zebasi.info": "{0} thrive on the warm savannah, cultivating the rich soil to provide food for their mighty population. \n\n{0} starts the game with the '{1}' tech.",
  "tribes.zebasi.news": "All profits from the Zebasi tribe is invested in solar energy projects.",
  "tribes.aimo": "Ai-Mo",
  "tribes.aimo.info": "The tranquil and wise {0} tribe inhabits the harshest, windiest and highest mountain range of the square, where they have found inner peace by meditating in the eternal evening light. \n\n{0} starts the game with the '{1}' tech.",
  "tribes.aquarion": "Aquarion",
  "tribes.aquarion.info": "From the deep oceans a long lost civilization appears! Their extreme isolation has given them special aquatic abilities that are unknown to the other more human tribes.\n\n{0} have a slightly different tech tree and have access to the unique amphibious turtle units that no other tribe can train.",
  "tribes.quetzali": "Quetzali",
  "tribes.quetzali.info": "The {0} tribe worship the bird gods of the red soil and live in harmony with the natural symmetry of their cubistic jungles. They are commonly seen riding huge flightless birds. \n\n{0} starts the game with the '{1}' tech.",
  "tribes.elyrion": "∑∫ỹriȱŋ",
  "tribes.elyrion.info": "The mysterious {0} defend their woodland homes with colorful magic and the ferocity of fire-breathing dragons! \n\nThey regard Nature as a holy spirit and cannot hunt animals or chop down trees, instead they start the game with the unique Enchantment tech that can turn regular animals into powerful monsters.",
  "tribes.yadakk": "Yădakk",
  "tribes.yadakk.info": "The {0} started out as a nomadic tribe in the beautiful harsh Khalee plains. Now they are the traders of the square, connecting their empire with impressive trade routes. \n\n{0} starts the game with the '{1}' tech.",
  "tribes.polaris": "Polaris",
  "tribes.polaris.info": "The {0} have lain trapped in the far reaches of the freezing tundra for eons, but were blessed by the unknowable Gaami with the power to expand their unnatural icy terrain farther than the weather permits. \n\nWith the might of sleds and Mamoos, the mystical {0} are determined to entomb the Square in ice and transform the land into a frigid paradise. \n\n{0} starts the game with the ability to freeze surrounding terrain using Moonies.",
  "building.capital.owner": "{0} is the {1} capital",
  "building.capital.owner.former": "{0} is the former capital of the {1} empire, currently occupied by {2} forces",
  "building.city.owner": "{0} is a city in the {1} empire",
  "building.village.owner": "This is a village that doesn't belong to any tribe",
  "building.ability.attract": "attracts a wild animal to a nearby forest tile every 3 turns",
  "building.produce": "produces {0} each turn",
  "building.produce.multiply.polaris": " ({0} for every {1} frozen tiles in the world)",
  "building.produce.multiply2.polaris": "Produces {0} for every {1} frozen tiles in the world",
  "building.produce.multiply": " ({0} for each nearby {1})",
  "building.produce.multiply2": "produces {0} for each nearby {1} each turn",
  "building.produce.multiply3": "produces {0} for each nearby {1}",
  "building.produce.reward.named": "produces {0} for {1}",
  "building.produce.reward": "produces {0}",
  "building.reward.tech": "gives you a {0}",
  "building.reward.instant": "instantly gives you {0}",
  "building.transform": "transforms a {0} into a {1}",
  "building.transform2": "transforms a {0} into a {1} with {2} ",
  "building.resource": "adds a {0} at the selected tile",
  "building.value": "is worth {0} points of score",
  "building.ability.embark": "Units that move here will turn into boats that can move on water.",
  "building.ability.route": "It creates trade routes through {0} to any other {1} within a 5 tiles radius.",
  "building.ability.route.and": " and ",
  "building.ability.road": "Build roads to connect cities with the capital. Connected cities get 1 population as a reward. Roads also give a movement bonus for all units.",
  "building.ability.ruin": "A ruin from an ancient civilization, it can contain valuable things! Go there with a unit to examine it.",
  "building.ability.patina": "{0} grows over time, so try to build them as early as possible.",
  "building.ability.limited": "You can only build one {0} per city.",
  "building.ability.unique": "{0} can only be built once.",
  "building.restriction.near": "Must be built next to a {0}.",
  "building.restriction.on": "Must be on a tile with {0}.",
  "building.names.city": "City",
  "building.names.ruin": "Ruin",
  "building.names.monument1": "Altar of Peace",
  "building.names.monument2": "Tower of wisdom",
  "building.names.monument3": "Grand Bazaar",
  "building.names.monument4": "Emperors Tomb",
  "building.names.monument5": "Gate of Power",
  "building.names.monument6": "Park of Fortune",
  "building.names.monument7": "Eye of God",
  "building.names.temple": "Temple",
  "building.names.burnforest": "Burn Forest",
  "building.names.road": "Road",
  "building.names.customshouse": "Customs House",
  "building.names.gather": "Harvest fruit",
  "building.names.farm": "Farm",
  "building.names.windmill": "Windmill",
  "building.names.fishing": "Fishing",
  "building.names.whalehunting": "Whale Hunting",
  "building.names.watertemple": "Water Temple",
  "building.names.port": "Port",
  "building.names.hunting": "Hunting",
  "building.names.clearforest": "Clear forest",
  "building.names.lumberhut": "Lumber Hut",
  "building.names.sawmill": "Sawmill",
  "building.names.growforest": "Grow Forest",
  "building.names.foresttemple": "Forest Temple",
  "building.names.mountaintemple": "Mountain Temple",
  "building.names.mine": "Mine",
  "building.names.forge": "Forge",
  "building.names.sanctuary": "Sanctuary",
  "building.names.enchant": "Enchant Animal",
  "building.names.enchant_whale": "Enchant Whale",
  "building.names.ice_bank": "Ice Bank",
  "building.names.iceport": "Outpost",
  "building.names.icetemple": "Ice Temple",
  "unit": "unit",
  "unit.info.attack": "Attack",
  "unit.info.defence": "Defence",
  "unit.info.movement": "Movement",
  "unit.info.health": "Health",
  "unit.info.range": "Range",
  "unit.info.skills": "Skills",
  "unit.names.giant": "Giant",
  "unit.names.crab": "Crab",
  "unit.names.egg": "Dragon Egg",
  "unit.names.wendy": "Gaami",
  "unit.names.bunny": "Bunny",
  "unit.names.scout": "Scout",
  "unit.names.boat": "Boat",
  "unit.names.warrior": "Warrior",
  "unit.names.rider": "Rider",
  "unit.names.knight": "Knight",
  "unit.names.defender": "Defender",
  "unit.names.ship": "Ship",
  "unit.names.battleship": "Battleship",
  "unit.names.catapult": "Catapult",
  "unit.names.archer": "Archer",
  "unit.names.priest": "Mind Bender",
  "unit.names.swordman": "Swordsman",
  "unit.names.amphibian": "Amphibian",
  "unit.names.tridention": "Tridention",
  "unit.names.dragon": "Baby Dragon",
  "unit.names.dragon_large": "Fire Dragon",
  "unit.names.polytaur": "Polytaur",
  "unit.names.seamonster": "Navalon",
  "unit.names.icemaker": "Mooni",
  "unit.names.battlesled": "Battle Sled",
  "unit.names.fortress": "Ice Fortress",
  "unit.names.icearcher": "Ice Archer",
  "unit.ability": "ability",
  "unit.abilities.dash": "DASH",
  "unit.abilities.escape": "ESCAPE",
  "unit.abilities.scout": "SCOUT",
  "unit.abilities.sneak": "SNEAK",
  "unit.abilities.hide": "HIDE",
  "unit.abilities.build": "BUILD",
  "unit.abilities.persist": "PERSIST",
  "unit.abilities.convert": "CONVERT",
  "unit.abilities.heal": "HEAL",
  "unit.abilities.swim": "SWIM",
  "unit.abilities.carry": "CARRY",
  "unit.abilities.grow": "GROW",
  "unit.abilities.fly": "FLY",
  "unit.abilities.splash": "SPLASH",
  "unit.abilities.decay": "DECAY",
  "unit.abilities.navigate": "NAVIGATE",
  "unit.abilities.freeze": "FREEZE",
  "unit.abilities.freezearea": "FREEZE AREA",
  "unit.abilities.autofreeze": "AUTO FREEZE",
  "unit.abilities.skate": "SKATE",
  "unit.abilities.fortify": "FORTIFY",
  "player.abilities.destroy": "destroy",
  "player.abilities.disband": "disband",
  "player.abilities.literacy": "literacy",
  "player.abilities.glide": "glide",
  "resource.names.fruit": "fruit",
  "resource.names.crop": "crop",
  "resource.names.fish": "fish",
  "resource.names.whale": "whale",
  "resource.names.game": "wild animal",
  "resource.names.metal": "metal",
  "terrain.unknown": "Unknown lands",
  "terrain.water": "Water",
  "terrain.ocean": "Ocean",
  "terrain.field": "Field",
  "terrain.forest": "Forest",
  "terrain.mountain": "Mountain",
  "terrain.ice": "Ice",
  "actionbox.building.level": "level {0}/{1}",
  "actionbox.tile.roads": "roads",
  "actionbox.city": "The city of {0}",
  "actionbox.city.level": "lvl {0}",
  "actionbox.village": "Village",
  "actionbox.unit.frozen": "Frozen {0}",
  "actionbox.unit.kills": "{0}/{1} kills",
  "actionbox.unit.veteran": "Veteran",
  "actionbox.unit.new": "Train a new {0} {1}",
  "actionbox.unit.ability": "Unit Ability",
  "actionbox.unit.train": "TRAIN",
  "actionbox.unit.upgrade": "UPGRADE",
  "actionbox.unit.toomany": "(too many)",
  "actionbox.unit.toomany.info": "This city can't support any more units. Upgrade the city to get more space for units.",
  "actionbox.building.doit": "DO IT",
  "actionbox.building.requiredtech": "You need to research {0} to do this, tap the 'Tech Tree' button.",
  "actionbox.building.techtree": "TECH TREE",
  "actionbox.insufficientfunds": "You don't have enough stars to buy this. Tap 'next turn' to get more stars.",
  "actionbox.confirm": "Confirm {0}",
  "actionbox.confirm.info": "Are you sure you want to do this?",
  "actionbox.confirm.button": "YES",
  "tooltip.tile.road": "Build a road to connect this city to your capital.",
  "tooltip.tile.choose_unit": "Choose a unit to produce.",
  "tooltip.tile.limit": "This city cannot support any more units.",
  "tooltip.tile.capture.enemy": "Your city is being captured by the enemy!",
  "tooltip.tile.capture": "This city is being captured.",
  "tooltip.tile.capture.tip": "Move a unit here to capture this city!",
  "tooltip.tile.produces": "Produces {0} each turn.",
  "tooltip.tile.level.polaris": "Freeze {0} more tiles to reach the next level",
  "tooltip.tile.level.next": "Next level in {0} turns",
  "tooltip.tile.level.max": "Has reached maximum level",
  "tooltip.tile.sailing": "Move a unit here to start sailing!",
  "tooltip.tile.monuments": "Monuments gives your empire extra score!",
  "tooltip.tile.ruin": "Move a unit here and examine these ancient ruins.",
  "tooltip.tile.blocked": "This resource is blocked by an enemy unit",
  "tooltip.tile.extract.upgrade": "Extract this resource to upgrade your city",
  "tooltip.tile.extract.convert": "This resource can be transformed into a unit",
  "tooltip.tile.extract.stars": "Extract this resource to earn stars instantly",
  "tooltip.tile.extract.research": "You need to research {0} to extract this resource",
  "tooltip.tile.outside": "This resource is outside of your empire",
  "tooltip.tile.research": "You need to research {0} to be able to move here",
  "tooltip.tile.explore": "Explore this area to see what it holds!",
  "tooltip.unit.city.capture": "Tap 'Capture' to add this city to your empire",
  "tooltip.unit.city.capture.next": "This city will be ready to capture next turn",
  "tooltip.unit.city.capture.flying": "Flying units cannot capture cities",
  "tooltip.unit.actions.none": "No actions left. Press 'Next Turn' to move this unit again.",
  "tooltip.unit.actions.move": "Click a blue mark to move.",
  "tooltip.unit.actions.attack": "Click a red mark to attack!",
  "tooltip.unit.enemy": "This is the enemy!",
  "tooltip.unit.enemy.territory": "This enemy is in your territory!",
  "tooltip.unit.enemy.city": "This enemy is capturing your city!",
  "tooltip.unit.grow.now": "Will grow to {0} at the end of this turn!",
  "tooltip.unit.grow.later": "Will grow to {0} in {1} turns.",
  "tooltip.unit.decay.now": "This unit will dissolve at the end of this turn.",
  "tooltip.unit.decay.later": "This unit will dissolve in {0} turns.",
  "tooltip.ability.disband": "Remove any of your own units and get half of its cost in return.",
  "tooltip.ability.destroy": "Remove any building within your borders, great for reconstructing your empire.",
  "tooltip.ability.literacy": "Reduce the price of all technologies by 20%.",
  "tooltip.ability.glide": "All non skating units get an extra move when moving on ice.",
  "tooltip.ability.dash": "This unit can attack after moving if there is an enemy in range.",
  "tooltip.ability.convert": "This unit can convert an enemy to your own tribe by attacking it.",
  "tooltip.ability.escape": "This unit can move again after attacking.",
  "tooltip.ability.persist": "This unit can keep attacking as long as it completely kills its victims.",
  "tooltip.ability.swim": "This unit is amphibious and can move on both land and water.",
  "tooltip.ability.carry": "This unit is carrying another unit inside.",
  "tooltip.ability.heal": "This unit can heal surrounding units.",
  "tooltip.ability.navigate": "This unit can move in any terrain even if you do not have the tech needed to move there.",
  "tooltip.ability.fly": "This unit can fly over any terrain with no movement penalties or bonuses.",
  "tooltip.ability.splash": "This unit causes splash damage to nearby units when attacking.",
  "tooltip.ability.grow": "This unit will eventually grow and become something else.",
  "tooltip.ability.sneak": "This unit can move past enemy units without stopping.",
  "tooltip.ability.scout": "This unit has double vision range.",
  "tooltip.ability.freeze": "This unit freezes its enemies when attacking them so they cannot move.",
  "tooltip.ability.freeze_area": "This unit can freeze the surrounding tiles including any enemy units.",
  "tooltip.ability.freeze_auto": "This unit freezes any surrounding tiles and units when moving.",
  "tooltip.ability.skate": "This unit gets double movement on ice tiles but on land their movement is limited to 1 tile and all other abilities are disabled.",
  "tooltip.ability.fortify": "This unit gets a defence bonus when defending in its own cities, double bonus with a city wall.",
  "buttons.ok": "OK",
  "buttons.exit": "EXIT",
  "buttons.save": "SAVE",
  "buttons.back": "BACK",
  "gameinfo.id": "id: {0}",
  "gameinfo.lastmove": "Last move: {0} ago",
  "gameinfo.updated": "updated: {0} ago",
  "gameinfo.turn": "Turn: {0}",
  "gameinfo.serverversion": "Server version: {0}",
  "gameinfo.gameover": "This game is over, open it to view the final result",
  "gameinfo.yourturn": "It is your turn to {0}",
  "gameinfo.opponentsturn": "Waiting for {0} to {1}",
  "gameinfo.start": "start the game",
  "gameinfo.picktribe": "pick tribe",
  "gameinfo.play": "play",
  "gamesaverbinary.unable.to.save": "I was unable to save the game, make sure you have sufficient storage on your device",
  "gamesaverbinary.unable.to.save.title": "Could not save :(",
  "gamesaverbinary.error.loading.moves": "Error loading moves",
  "polyplayer.task": "task",
  "polyplayer.task.explorer.title": "Explorer",
  "polyplayer.task.explorer.description": "Explore every single tile in this square world",
  "polyplayer.task.war.title": "No mercy",
  "polyplayer.task.war.description": "Wipe out an enemy",
  "polyplayer.task.pacifist.title": "Pacifist",
  "polyplayer.task.pacifist.description": "Make no attacks for 5 turns",
  "polyplayer.task.killer.title": "Killer",
  "polyplayer.task.killer.description": "Kill 10 enemies in battle",
  "polyplayer.task.wealth.title": "Wealth",
  "polyplayer.task.wealth.description": "Gather 100 stars",
  "polyplayer.task.genius.title": "Genius",
  "polyplayer.task.genius.description": "Discover all available technologies",
  "polyplayer.task.metropolis.title": "Metropolis",
  "polyplayer.task.metropolis.description": "Create a level 5 city",
  "polyplayer.task.network.title": "Network",
  "polyplayer.task.network.description": "Connect 5 cities to your capital",
  "task.info": "{0} to get {1}",
  "price.stars": "star",
  "price.stars.plural": "stars",
  "price.population": "population",
  "price.population.plural": "population",
  "price.points": "point",
  "price.points.plural": "points",
  "wcontroller.online.yourturn.title": "It's your turn!",
  "wcontroller.online.yourturn.description": "Tap ok to continue the game when you are ready.",
  "wcontroller.convertvillage.description": "The villagers agree to join your emerging empire!",
  "wcontroller.convertvillage.title": "Village converted!",
  "wcontroller.capital.regained.description": "You have regained control of your capital, trade networks are reestablished",
  "wcontroller.capital.regained.title": "Great news!",
  "wcontroller.capital.lost.description": "Your capital has been captured by the {0} hordes! All your trade connections are cancelled until you regain control of your capital",
  "wcontroller.capital.lost.title": "Bad news!",
  "wcontroller.capital.captured.description": "You have captured the {0} capital! All their trade connections are cancelled until they regain control of their capital",
  "wcontroller.capital.captured.title": "Great news!",
  "wcontroller.capital.captured2.description": "{0} is now part of the {1} empire",
  "wcontroller.capital.captured2.title": "City captured!",
  "wcontroller.kill.upgrade.description": "Unit is ready to upgrade!",
  "wcontroller.kill.upgrade.title": "Level Up!",
  "wcontroller.examine.water.elyrion": "You encountered an enchanted {0} who joined your tribe!",
  "wcontroller.examine.water": "You encountered a band of friendly pirates who joined your tribe!",
  "wcontroller.examine.water.title": "Battle ship",
  "wcontroller.examine.giant": "You found a friendly {0} who joined your tribe!",
  "wcontroller.examine.explorer": "You meet some locals who show you the surrounding lands.",
  "wcontroller.examine.explorer.title": "Explorer",
  "wcontroller.examine.tech": "You found some old scrolls containing the secret of {0}.",
  "wcontroller.examine.tech.title": "Scrolls of Wisdom",
  "wcontroller.examine.stars": "The ancient ruins are filled with valuable resources!",
  "wcontroller.examine.stars.title": "Resources",
  "wcontroller.examine.population": "You meet a nomadic tribe that settle in your capital!",
  "wcontroller.examine.population.title": "Population",
  "wcontroller.move.unto.unit": "CANT MOVE ONTO ANOTHER UNIT",
  "wcontroller.building.upgrade": "Your {0} upgraded to a {1}!",
  "wcontroller.building.upgrade.reward": "{0} upgraded to level {1} and its production has increased +1. You also get to pick an additional reward:",
  "wcontroller.reward.workshop": "workshop",
  "wcontroller.reward.citywall": "city wall",
  "wcontroller.reward.populationgrowth": "population growth",
  "wcontroller.reward.park": "park",
  "wcontroller.reward.explorer": "explorer",
  "wcontroller.reward.resources": "resources",
  "wcontroller.reward.bordergrowth": "border growth",
  "wcontroller.reward.superunit": "super unit",
  "wcontroller.unit.promotion": "Your unit has gained veteran status! Health increased.",
  "wcontroller.unit.promotion.title": "Unit Promoted!",
  "wcontroller.meet.tribe.leader": "Their leader",
  "wcontroller.meet.tribe.bigger.hostile": "laughs at your puny excuse of a Tribe.",
  "wcontroller.meet.tribe.bigger.friendly": "salutes you friendly but pays no attention to your tiny realm.",
  "wcontroller.meet.tribe.smaller.hostile": "seems a bit hostile and greets you suspiciously.",
  "wcontroller.meet.tribe.smaller.friendly": "bows in awe of your might mighty civilization.",
  "wcontroller.meet.tribe.tech.hostile": "You manage to steal the secret of {0}!",
  "wcontroller.meet.tribe.tech.friendly": "As a gesture of good will they share the secret of {0}!",
  "wcontroller.meet.tribe.resource.hostile": "You manage to steal some valuable gold pieces!",
  "wcontroller.meet.tribe.resource.friendly": "They present you with a gift of valuable resources!",
  "wcontroller.tribe.destroy": "You have destroyed {0}!",
  "wcontroller.tribe.destroy.title": "Blood!",
  "wcontroller.tribe.destroy2": "{0} has been destroyed by the {1}!",
  "wcontroller.tribe.destroy.all": "You have destroyed all opposing tribes and unified the entire square under your command!",
  "wcontroller.tribe.destroy.all.title": "Domination!",
  "wcontroller.city.disconnect": "{0} was disconnected from {1}",
  "wcontroller.city.disconnect.title": "Trade route lost!",
  "wcontroller.turn.end": "Ending turn...",
  "wcontroller.turn.saving": "Saving game to server...",
  "wcontroller.turn.notification": "It's your turn in {0} (turn {1})",
  "wcontroller.turn.passed": "Game passed on to {0}",
  "wcontroller.turn.passed.title": "Turn complete",
  "wcontroller.turn.error": "Could not reach the multiplayer server. Please make sure you are connected to the internet and try again.",
  "wcontroller.turn.error.title": "Network Error",
  "wcontroller.turn.next": "Next up",
  "wcontroller.load.error": "There is no saved game to resume, start a new game!",
  "wcontroller.load.error.title": "No saved game",
  "wcontroller.load.notpartof": "You are not a part of this game",
  "wcontroller.load.wait": "Wait until this game has fully downloaded before opening it.",
  "wcontroller.load.wait.title": "Still loading...",
  "wcontroller.load.update": "This game is using a newer version of Polytopia, you need to head over to {0} and update it before you can play.",
  "wcontroller.load.update.title": "Update Required",
  "wcontroller.removingplayer": "Removing Player",
  "wcontroller.not.your.turn": "Sorry, it's not your turn yet!",
  "technology.intro": "This tech will enable the following:",
  "technology.build": "{0} makes it possible to build {1}",
  "technology.movement": "Movement",
  "technology.movement.info": "enables movement in {0}",
  "technology.defence": "Defence Bonus",
  "technology.defence.info": "gives your unit extra strength when defending in {0}",
  "technology.task": "{0} activates the {1} task",
  "technology.ability": "ability",
  "technology.ability.info": "{1} gives you the ability to {1}",
  "technology.names.basic": "basic",
  "technology.names.riding": "Riding",
  "technology.names.freespirit": "Free Spirit",
  "technology.names.chivalry": "Chivalry",
  "technology.names.roads": "Roads",
  "technology.names.trade": "Trade",
  "technology.names.organization": "Organization",
  "technology.names.shields": "Shields",
  "technology.names.farming": "Farming",
  "technology.names.construction": "Construction",
  "technology.names.fishing": "Fishing",
  "technology.names.whaling": "Whaling",
  "technology.names.aquatism": "Aquatism",
  "technology.names.sailing": "Sailing",
  "technology.names.navigation": "Navigation",
  "technology.names.hunting": "Hunting",
  "technology.names.forestry": "Forestry",
  "technology.names.mathematics": "Mathematics",
  "technology.names.archery": "Archery",
  "technology.names.spiritualism": "Spiritualism",
  "technology.names.climbing": "Climbing",
  "technology.names.meditation": "Meditation",
  "technology.names.philosophy": "Philosophy",
  "technology.names.mining": "Mining",
  "technology.names.smithery": "Smithery",
  "technology.names.freediving": "Free Diving",
  "technology.names.spearing": "Spearing",
  "technology.names.forestmagic": "Forest Magic",
  "technology.names.watermagic": "Water Magic",
  "technology.names.frostwork": "Frostwork",
  "technology.names.polarwarfare": "Polar Warfare",
  "technology.names.polarism": "Polarism",
  "techview.info": "Tech costs increase for each city in your empire.",
  "techview.info.literacy": "Literacy reduce the price of all technologies by 20%!",
  "techview.locked": "(locked)",
  "techview.locked.info": "You must research {0} before you can learn {1}.",
  "techview.completed": "(completed)",
  "techview.completed.info": "You have already researched this technology.",
  "techview.expensive.info": "You don't have enough stars to buy this. Tap 'next turn' to get more stars.",
  "techview.research": "RESEARCH",
  "action.info.attack": "make an attack with this unit. Select the unit and tap any of the RED targets if you want to attack",
  "action.info.recover": "Recover",
  "action.info.healothers": "Heal Others",
  "action.info.train": "train a unit in this city. Units can be used to explore the world and to attack enemies and defend your cities",
  "action.info.move": "move this unit. Select the unit and tap any of the BLUE targets",
  "action.info.capture": "capture",
  "action.info.capture2": "capture this city. Cities generate stars every turn that you can use to develop your empire",
  "action.info.destroy": "destroy",
  "action.info.disband": "Disband {0}",
  "action.info.remove": "Remove",
  "action.info.cityreward": "City Reward",
  "action.info.reward": "Reward",
  "action.info.trip": "Trip",
  "action.info.meet": "Meet",
  "action.info.promote": "Promote",
  "action.info.examine": "Examine",
  "action.info.endturn": "end this turn to get more resources and unit moves. Press the 'Next Turn' button",
  "action.info.stay": "Stay",
  "action.info.healarea": "healArea",
  "action.info.freezearea": "Freeze Area",
  "action.info.breakice": "Break Ice",
  "action.info.do": "do {0} here",
  "action.info.build": "build a {0} here",
  "action.info.reward.population": "That will increase the population of the nearest city. When the population becomes big enough the city will level up and produce more resources",
  "action.info.reward.resources": "That will give you an instant reward of {0} resources",
  "action.info.research": "research {0}.",
  "actionbtn.upgrade": "Upgrade to {0}",
  "actionbtn.remove.building": "Building",
  "actionbtn.remove.roads": "Roads",
  "stringtools.typelist.and": "and",
  "topbar.score": "Score",
  "topbar.turn": "Turn",
  "topbar.stars": "Stars (+{0})",
  "bottommenu.gamestats": "Game Stats",
  "bottommenu.menu": "Menu",
  "bottommenu.nextturn": "Next Turn",
  "bottommenu.techtree": "Tech Tree",
  "endscreen.done": "DONE",
  "endscreen.ruledby": "Ruled by {0}",
  "endscreen.army&territory": "Army & Territory",
  "endscreen.monuments&temples": "Monuments & temples",
  "endscreen.cities": "Cities",
  "endscreen.science": "Science",
  "endscreen.units": "{0} units, {1} tiles empire",
  "endscreen.culture": "{0} monuments, {1} temples",
  "endscreen.citiescount": "{0} cities",
  "endscreen.techscore": "{0}/{1} technologies researched",
  "endscreen.bonus": "Difficulty Bonus",
  "endscreen.finalscore": "FINAL SCORE",
  "endscreen.speedskills": "Speed skills",
  "endscreen.domination.win": "{0}/{1} turns",
  "endscreen.domination.loss": "{0} turns",
  "endscreen.battle": "Battle skills",
  "endscreen.battle.info": "{0} units lost",
  "endscreen.destroyed": "Tribes destroyed",
  "endscreen.destroyed.info": "{0}/{1}",
  "endscreen.rating": "Difficulty rating",
  "endscreen.finalrating": "FINAL RATING",
  "endscreen.nextstar.percent": "{0}% needed for next star",
  "endscreen.nextstar": "{0} needed for next star",
  "endscreen.topresult": "New top result!",
  "endscreen.topresult.title": "Great!",
  "endscreen.personal": "New personal all time high!",
  "endscreen.personal.title": "Amazing!",
  "endscreen.showhiscore": "SHOW HISCORE",
  "endscreen.winner": "{0} win!",
  "endscreen.victory": "VICTORY",
  "endscreen.gameover": "GAME OVER",
  "highscore.title": "HIGH SCORE",
  "highscore.today": "Today",
  "highscore.thisweek": "This Week",
  "highscore.alltime": "All Time",
  "highscore.alltribes": "All Tribes",
  "highscore.hiscore": "hiscore",
  "highscore.loading": "Loading..",
  "highscore.notavailable": "High Score not available.",
  "multiplayer.passplay": "Pass & Play",
  "multiplayer.passplay.info": "Challenge your friends in a multiplayer match on the same device. Just pass it on to the next player when your turn is done.",
  "multiplayer.activegames": "Active Games",
  "multiplayer.finishedgames": "Finished Games",
  "multiplayer.creategame": "Create game",
  "multiplayer.clipboard": "Gamedata added to clipboard",
  "multiplayer.clipboard.title": "Voilà!",
  "gamestats.gamemode": "Game mode: {0}",
  "gamestats.bonus": "Difficulty bonus: {0}",
  "gamestats.speed": "Speed skills",
  "gamestats.speed.info": "{0}/{1} turns",
  "gamestats.battle": "Battle skills",
  "gamestats.battle.info": "{0} won, {1} lost",
  "gamestatus.tribes": "Tribes destroyed",
  "gamestatus.difficulty": "Difficulty rating",
  "gamestatus.capitals": "Capitals Owned",
  "gamestatus.scores": "Scores",
  "gamestatus.ruled": "Ruled by {0}",
  "gamestatus.ruled.you": "Ruled by you",
  "gamestatus.unknown.tribe": "Unknown tribe",
  "gamestatus.unknown.ruler": "Unknown ruler",
  "gamestatus.score": "score: {0} points",
  "gamestatus.city": "{0} city",
  "gamestatus.cities": "{0} cities",
  "gamestatus.destroyed": "destroyed",
  "gamestatus.tasks": "Tasks {0}/{1}",
  "gamestatus.tasks.complete": "Complete!",
  "settings.title": "SETTINGS",
  "settings.volume": "Audio volume {0}",
  "settings.soundeffects": "Sound Effects",
  "settings.ambience": "Ambience",
  "settings.tribemusic": "Tribe Music",
  "settings.suggestions": "Suggestions",
  "settings.info": "Info on Build",
  "settings.confirm": "Confirm turn",
  "settings.saveexit": "EXIT TO MENU",
  "settings.on": "ON",
  "settings.off": "OFF",
  "settings.language": "Language",
  "settings.restartlanguage": "Please restart Polytopia to fully switch language",
  "settings.language.load.title": "Custom language (BETA)",
  "settings.language.load.info": "Load a Polytopia language file from a remote server using https. This is an experimental feature, not for the faint hearted.",
  "settings.language.load.input": "Language file url:",
  "settings.language.load.button": "LOAD",
  "throne.title": "THRONE ROOM",
  "throne.reset": "reset scores",
  "throne.playerinfo": "Player Info",
  "throne.playerid": "Player ID",
  "throne.clipboard": "Player ID added to clipboard",
  "throne.clipboard.title": "Voilà!",
  "throne.alias": "Alias",
  "throne.played": "Games Played",
  "throne.topscore": "Top Score",
  "throne.toprating": "Top Rating",
  "throne.resetwarning": "Are you sure you want to reset all your saved scores and ratings? This can not be undone.",
  "throne.resetwarning.title": "Reset Scores",
  "throne.reset.complete": "Scores are now reset",
  "throne.google.achievements": "ACHIEVEMENTS",
  "throne.google.signedin": "You are signed in using Google Play",
  "throne.google.out": "SIGN OUT",
  "throne.google.info": "(!)You need to sign in with Google Play to save your game data and high scores.",
  "throne.google.in": "SIGN IN",
  "consent.approval.title": "Welcome to Polytopia!",
  "consent.approval.info": "To improve Polytopia with usage statistics and to store your high scores and settings we need your approval to access to some personal data.\nYou can read more about what data we collect in our <u><a href='{0}'>privacy policy</a></u>.\n(You can revoke your approval at any time in the 'Throne Room')",
  "consent.approve": "APPROVE",
  "consent.deny": "DENY",
  "consent.enabled": "Personal Data Enabled",
  "consent.enabled.info": "You currently allow Polytopia to access some personal data to improve service and store your high scores etc. For more info, read our <u><a href='{0}'>privacy policy</a></u>.",
  "consent.disabled": "Personal Data Disabled",
  "consent.disabled.info": "To save personal data like high scores and usage statistics we need you approval. For more info, read our <u><a href='{0}'>privacy policy</a></u>.",
  "consent.revoke": "revoke consent",
  "onlineview.title": "MULTIPLAYER",
  "onlineview.loadingservice": "Loading Service",
  "onlineview.yourturn": "Your Turn",
  "onlineview.theirturn": "Their Turn",
  "onlineview.reloading": "Reloading..",
  "onlineview.reloading.release": "Release to reload...",
  "onlineview.newgame": "New Game",
  "onlineview.friends": "Friends",
  "onlineview.profile": "Profile",
  "onlineview.passplay": "Pass & Play",
  "onlineview.refresh": "Refresh",
  "onlineview.profile.available": "Profile is only available when connected to the server.",
  "onlineview.friendlist.available": "Friend List is only available when connected to the server.",
  "onlineview.servicedisabled": "The multiplayer server is temporarily disabled, please try again later. Also make sure that you are using the latest version of the Battle of Polytopia.",
  "onlineview.loaderror": "Could not connect to the multiplayer server. Check your internet connection and try again.",
  "onlineview.uptodate": "All games are up to date",
  "onlineview.intro.fix": "Hi {0}! \n\nTo play online multiplayer there are a few things you need to fix:",
  "onlineview.intro.update": "Online multiplayer is not available, please update Polytopia to the latest version",
  "onlineview.gameinvitations": "Game Invitations",
  "onlineview.nogames.intro": "Hi {0}!",
  "onlineview.nogames.start": "Let's get some online games going with your friends. Tap 'New game' to start one.",
  "onlineview.nogames.first": "The first thing you need is to connect with some other humans who play Polytopia. Tap 'Friends' to add them.",
  "onlineview.or": "or",
  "onlineview.passplay.start": "Start a local Pass & Play game by tapping 'New Game'",
  "onlineview.login.ios": "Log in to {0}",
  "onlineview.login.ios.info": "Using {0} ensures that you have a unique and persistent Player ID that will keep your data safe across devices.",
  "onlineview.notifications": "Enable notifications",
  "onlineview.notifications.info": "We use notifications to communicate the status of your ongoing multiplayer games.",
  "onlineview.purchase": "Purchase one Tribe",
  "onlineview.purchase.info": "Running an online multiplayer service costs real money and we rely solely on the kind support from players like you.",
  "onlineview.completed": "completed",
  "onlineview.required": "required",
  "onlineview.check": "check",
  "onlineview.fixit": "FIX IT",
  "onlineview.clipboard": "Game ID added to clipboard: {0}",
  "onlineview.clipboard.title": "Voilà!",
  "onlineview.game.join": "JOIN GAME",
  "onlineview.game.start": "START GAME",
  "onlineview.game.open": "OPEN",
  "onlineview.game.size": "Map Size",
  "onlineview.game.moreinfo": "More info",
  "onlineview.game.gameinfo": "Game Info",
  "onlineview.game.you": "You",
  "onlineview.game.resign": "RESIGN",
  "onlineview.game.decline": "DECLINE",
  "onlineview.game.delete": "DELETE",
  "onlineview.game.resign.title": "Resign",
  "onlineview.game.resign.info": "Are you sure you want to leave this game permanently?",
  "onlineview.game.old.title": "Old file version",
  "onlineview.game.old.info": "This game was created with an outdated version of the game. It can unfortunately not be loaded :( My suggestion is that you delete it and start a new one. Sorry for the inconvenience.",
  "onlineview.game.player.left": "{0} has left the game {1} and has been replaced by a bot.",
  "onlineview.game.player.kicked": "You have been removed from game {0}",
  "onlineview.game.player.invited": "You are invited to a new game, {0}",
  "firebaseservice.status.connecting": "Connecting to the Polytopia server...",
  "firebaseservice.status.loading": "Loading game data...",
  "firebaseservice.status.loading.count": "Loading game data, {0} left",
  "firebaseservice.status.loading.player": "Loading player data...",
  "firebaseservice.status.sync": "Sync chronometer..",
  "firebaseservice.status.sync.player": "Syncing your player data..",
  "firebaseservice.status.checking": "Checking for changes..",
  "firebaseservice.status.loading.messages": "Loading messages...",
  "firebaseservice.important.title": "Important information",
  "firebaseservice.important.deleted": "{0} has deleted you from multiplayer game {1}",
  "firebaseservice.error": "There was an error saving the game. Please try again.",
  "firebaseservice.invite": "You are invited to a new game, {0}",
  "firebaseservice.removed": "You have been removed from game {0}",
  "friendlist.title": "FRIEND LIST",
  "friendlist.new.caps": "NEW FRIEND",
  "friendlist.new.title": "Add a new friend",
  "friendlist.new.info": "Enter the player ID of your friend. (they can find it on this Friend page on their device)",
  "friendlist.new.button": "ADD",
  "friendlist.new.input": "Player ID:",
  "friendlist.new.myself.title": "Me, myself & I",
  "friendlist.new.myself.info": "Seems like you tried to add yourself as a friend. That might sound like a nice thing to do but it would add an existential layer to the game that we cannot handle at the moment. Please submit a player ID of someone else.",
  "friendlist.new.empty.title": "Emptiness",
  "friendlist.new.empty.info": "The player ID you entered was completely empty! You should not try to make friends with the void, it is a very lonely path.",
  "friendlist.new.exists.title": "Duplicate Player",
  "friendlist.new.exists.info": "You are already friends with {0}",
  "friendlist.new.looking.title": "Loading Player",
  "friendlist.new.looking.info": "Looking for player {0}",
  "friendlist.new.added.title": "Player Added",
  "friendlist.new.added.info": "Added player {0}",
  "friendlist.new.error.title": "Error loading player",
  "friendlist.new.error.info": "Could not find player with the ID {0}.",
  "friendlist.new.error2.title": "Player not found",
  "friendlist.new.error2.info": "Could not find any player with the ID {0}. Error: {1}",
  "friendlist.loading": "Loading friends...",
  "friendlist.error": "Error loading friends",
  "friendlist.friends": "Friends",
  "friendlist.friends.old": "Outdated friends",
  "friendlist.local": "Local Players",
  "friendlist.bots": "Bots",
  "friendlist.bot": "{0} Bot",
  "friendlist.player": "Player {0}",
  "friendlist.remove": "REMOVE",
  "friendlist.reload": "RELOAD",
  "friendlist.checking": "Checking friend status..",
  "friendlist.friend.update": "{0} needs to update to the latest version of Polytopia before you can invite them to new games.",
  "friendlist.friend.updated": "{0} is now on the new server",
  "friendlist.friend.notupdated": "{0} is still on the old server",
  "friendlist.removed.title": "Player Removed",
  "friendlist.removed.info": "Removed player {0}",
  "idconsole.playerid": "Your Player ID:",
  "idconsole.share": "Send this Player ID to anyone you want to play against. Tap it to copy.",
  "idconsole.clipboard": "Player ID {0} added to clipboard",
  "idconsole.clipboard.title": "Voilà!",
  "playerpickerview.title": "PICK PLAYERS",
  "playerpickerview.name": "Game Name",
  "playerpickerview.startgame": "START GAME",
  "playerpickerview.addplayer": "ADD PLAYER",
  "playerpickerview.size": "Map size: {0} tiles",
  "playerpickerview.mode": "Game mode: {0}",
  "playerpickerview.players": "Players ({0}/{1})",
  "playerpickerview.you": "{0} (you)",
  "playerpickerview.bot": "{0} ({1} bot)",
  "playerpickerview.human": "No human player",
  "playerpickerview.human.info": "There needs to be at least one human player to start a game",
  "gameitem.join": "Join this game or decline the invitation",
  "gameitem.join.wait": "Waiting for {0} to pick tribe",
  "gameitem.ready": "Ready to start!",
  "gameitem.ready.wait": "{0} can start the game",
  "gameitem.turn.your": "Your turn",
  "gameitem.turn.other": "Waiting for {0}",
  "gameitem.gameover": "This game is over, tap to see the end.",
  "gameitem.pick": "{0}, pick your tribe",
  "gameitem.start": "{0}, start the game",
  "gameitem.turn": "{0}, take your turn",
  "gameitem.ended": "This game is over.",
  "gameitem.pending": "Pending. Open and save to server.",
  "gameitem.timeup": "Time Up!",
  "gameitem.timeup.info": "Your time to make a move is up, do you want to resign?",
  "gameitem.timelimit": "Time limit",
  "gameitem.timelimit.info": "You have {0} to make your move, after that you will be removed from the game.",
  "gameitem.kick": "Duh!",
  "gameitem.kick.info": "Do you want to kick {0} out of from this game? A bot will take control of the tribe.",
  "gameitem.kick.action": "KICK",
  "gameitem.slow": "Come on..",
  "gameitem.slow.info": "{0} has {1} to make a move. Send a rude reminder to make {0} hurry up?",
  "gameitem.slow.action": "REMIND",
  "gameitem.timeleft": "Cool runnings",
  "gameitem.timeleft.info": "{0} still has {1} to make a move.",
  "gameitem.reload": "RELOAD",
  "gameitem.remind.max": "That's enough",
  "gameitem.remind.max.info": "Reminder already sent to {0}",
  "gameitem.remind.notification": "We are waiting for you to play in {0}. Come on!!",
  "gameitem.remind.notify": "Done",
  "gameitem.remind.notify.info": "Reminder sent to {0}",
  "mplayerstats.title": "PLAYER PROFILE",
  "mplayerstats.clear": "clear data",
  "mplayerstats.reload": "reload multiplayer data",
  "mplayerstats.multiplayer.faq": "MULTIPLAYER FAQ",
  "mplayerstats.alias": "Alias",
  "mplayerstats.friends": "nº of friends",
  "mplayerstats.games": "Games Played",
  "mplayerstats.server": "Server version",
  "mplayerstats.lost": "Feeling lost? Check the:",
  "credits.title": "ABOUT",
  "credits.subtitle": "Indie Delight",
  "credits.midjiwan": "The Battle of Polytopia is constantly being created by Midjiwan, a tiny indie game studio in Stockholm, Sweden.",
  "credits.learnmore1": "Want to know everything?",
  "credits.learnmore2": "Check the extensive Wikia database created by the Polytopia community:",
  "credits.wiki": "POLYTOPIA WIKI",
  "credits.showsupport": "Show your support!",
  "credits.getmerch": "Get some official Polytopian merch:",
  "credits.visitshop": "VISIT THE SHOP",
  "credits.programming": "Programming & Design",
  "credits.sound": "Sound & Music",
  "credits.tribe": "Tribe Design",
  "credits.debuginfo": "Debug info",
  "credits.build": "Build version",
  "credits.os": "OS",
  "credits.screendpi": "Screen DPI",
  "credits.screensize": "Screen size",
  "credits.privacy": "Privacy Policy",
  "credits.terms": "Terms of Service",
  "date.day": "day",
  "date.days": "days",
  "date.hour": "hour",
  "date.hours": "hours",
  "date.minute": "minute",
  "date.minutes": "minutes",
  "date.second": "second",
  "date.seconds": "seconds",
  "friendlist.friendrequests": "Friend Requests",
  "friendlist.pendinginvitations": "Pending Invitations",
  "friendlist.rejectedinvitations": "Rejected Invitations",
  "friendlist.new.Search": "SEARCH",
  "friendlist.new.accept": "ACCEPT",
  "friendlist.new.reject": "REJECT",
  "friendlist.new.Results": "Results:",
  "friendlist.requestrecieved": "Friend Request Received",
  "friendlist.requestaccepted": "Friend Request Accepted",
  "mplayerstats.appversion": "App version:",
  "gameitem.start.wait": "Waiting for {0} to start the game",
  "gamesettings.disableTribes": "Enable or Disable Tribes",
  "startmenu.quit": "QUIT",
  "buttons.quit": "Quit",
  "mplayerstats.reloading": "Reloading..",
  "mplayerstats.uptodate": "Everything is up to date",
  "mplayerstats.loaderror": "Could not connect to the multiplayer server. Check your internet connection and try again.",
  "friendlist.reloading": "Reloading..",
  "friendlist.uptodate": "Everything is up to date",
  "friendlist.loaderror": "Could not connect to the multiplayer server. Check your internet connection and try again.",
  "friendlist.cancelinvite.button": "CANCEL INVITE",
  "friendlist.acceptinvite.button": "ACCEPT INVITE",
  "friendlist.friendshipstate": "Friendship status: {0}",
  "friendlist.friendshipstate.none": "None.",
  "friendlist.friendshipstate.isyou": "This is you.",
  "friendlist.friendshipstate.accepted": "Friend request accepted. You are both friends.",
  "friendlist.friendshipstate.sentrequest": "Friend request sent. Waiting for response.",
  "friendlist.friendshipstate.receivedrequest": "Friend request received. Do you accept?",
  "friendlist.friendshipstate.rejected": "Friend request rejected."
}
